src/main/resources/hudson/views/UserRelevanceFilter/config.jelly

```diff
@@ -4,46 +4,66 @@
 	xmlns:i="jelly:fmt" xmlns:p="/lib/hudson/project">
   <f:entry title="${%Search name}:">
     <div>
-    <f:checkbox name="matchUserId" field="matchUserId" default="true" /> ${"%Search for current user's id"}<br/>
-    <f:checkbox name="matchUserFullName" field="matchUserFullName" default="true" /> ${"%Search for current user's full name"} <br/>
+      <div>
+        <f:checkbox title="${%Search for current user's id}" field="matchUserId" default="true"/>
+      </div>
+      <div>
+        <f:checkbox title="${%Search for current user's full name}" field="matchUserFullName" default="true"/>
+      </div>
     </div>
-    <div style="margin-top:10px">
-    <f:checkbox name="ignoreCase" field="ignoreCase" default="true" /> ${%Ignore case} <br/>
-    <f:checkbox name="ignoreWhitespace" field="ignoreWhitespace" default="true" /> ${%Ignore whitespace} <br/>
-    <f:checkbox name="ignoreNonAlphaNumeric" field="ignoreNonAlphaNumeric" default="true" /> ${%Ignore non-alphanumeric} <br/>
+    <div class="jenkins-!-margin-top-2">
+      <div>
+        <f:checkbox title="${%Ignore case}" field="ignoreCase" default="true"/>
+      </div>
+      <div>
+        <f:checkbox title="${%Ignore whitespace}" field="ignoreWhitespace" default="true"/>
+      </div>
+      <div>
+        <f:checkbox title="${%Ignore non-alphanumeric}" field="ignoreNonAlphaNumeric" default="true"/>
+      </div>
     </div>
   </f:entry>
   <f:entry title="${%Match jobs}:">
-    <f:checkbox name="matchEmail" field="matchEmail" default="true" /> ${%Match jobs where user is email recipient} <br/>
-    <f:checkbox name="matchBuilder" field="matchBuilder" default="true" onclick="showOrHideBuildOptions(this)"/> ${%Match jobs with builds "started by" user} <br/>
-    <f:checkbox name="matchScmChanges" field="matchScmChanges" default="true" onclick="showOrHideBuildOptions(this)"/> ${%Match jobs with builds where user is source code committer}<br/>
+    <div>
+      <f:checkbox title="${%Match jobs where user is email recipient}" field="matchEmail" default="true"/>
+    </div>
+    <div>
+      <f:checkbox title="${%Match jobs with builds 'started by' user}" name="matchBuilder" field="matchBuilder" default="true"
+                  onclick="showOrHideBuildOptions(this)"/>
+    </div>
+    <div>
+      <f:checkbox title="${%Match jobs with builds where user is source code committer}" name="matchScmChanges" field="matchScmChanges"
+                  default="true" onclick="showOrHideBuildOptions(this)"/>
+    </div>
     <div class="showOrHideBuildOptions">
     <script>
       function showOrHideBuildOptions(elem) {
-        var tr1 = elem.parentElement.parentElement;
-        var matchBuilder = tr1.querySelector('[name="matchBuilder"]');
-        var matchScmChanges = tr1.querySelector('[name="matchScmChanges"]');
+        var jobFilter = elem.closest('[name="jobFilters"]');

```

Comment:
use closest to get the job filter rather than silly parent element guessing that core can change at any point...